main.go

```diff
@@ -139,10 +143,15 @@ func initComponents(_ context.Context, p cluster.Platform) error {
 	if err := datasciencepipelinesctrl.Init(p); err != nil {
 		multiErr = multierror.Append(multiErr, err)
 	}
-
 	if err := kueuectrl.Init(p); err != nil {
 		multiErr = multierror.Append(multiErr, err)
 	}
+	if err := mmctrl.Init(p); err != nil {
+		return err
+	}
+	if err := mmctrl.Init(p); err != nil {
```

Comment:
seems a duplication, probably you meant `mcctrl`

Reply:
yes, came after rebase , fixed now